riff-raff/app/ci/ContinuousDeployment.scala

```diff
@@ -48,7 +48,10 @@ class ContinuousDeployment(deployments: Deployments) extends Lifecycle with Logg
       if (!ChangeFreeze.frozen(params.stage.name)) {
         log.info(s"Triggering deploy of ${params.toString}")
         try {
-          deployments.deploy(params)
+          deployments.deploy(params, requestSource = ContinuousDeploymentRequestSource) match {
```

Comment:
Would a `leftMap` (https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/syntax/either.scala#L120) be more readable here, or would that make you think we cared about the result value?